extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -191,66 +191,64 @@ private function fetchAll($useMasterDb = false) {
 				}
 				*/
 
-				if($row->type == BADGE_TYPE_INTRACKEDITPLUSCATEGORY)
-					$this->mInTrackEditPlusCategory[$row->id] = array('enabled' => $row->enabled, 'category' => $row->cat);
-				elseif($row->type == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM) {
-					$this->mNotInTrackCommunityPlatinum[$row->id] = array(
+				if( $row->type == BADGE_TYPE_INTRACKEDITPLUSCATEGORY )
+					$this->mInTrackEditPlusCategory[ $row->id ] = array( 'enabled' => $row->enabled, 'category' => $row->cat );
+				elseif( $row->type == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM ) {
+					$this->mNotInTrackCommunityPlatinum[ $row->id ] = array(
 						'enabled' => $row->enabled,
 						'is_sponsored' => $row->sponsored,
 						'badge_tracking_url' => $row->badge_tracking_url,
 						'hover_tracking_url' => $row->hover_tracking_url,
 						'click_tracking_url' => $row->click_tracking_url
 					);
 				} else {
-					wfDebug("Unknown badge type for {$row->id}");
-					wfProfileOut(__METHOD__);
+					wfDebug( "Unknown badge type for {$row->id}" );
+					wfProfileOut( __METHOD__ );
 					return false;
 				}
 
 				//flagging cache
 				//$this->mLoadedData[$row->id] = true;
 			}
 
-			$dbr->freeResult($res);
+			$dbr->freeResult( $res );
 			$this->mAllDataFetched = true;
 		}
 
-		wfProfileOut(__METHOD__);
+		wfProfileOut( __METHOD__ );
 
 		return true;
 	}
 
-	private function fetchOne($badgeTypeId) {
-		wfProfileIn(__METHOD__);
+	private function fetchOne( $badgeTypeId ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "fetchOne" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1144)